src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/DiscoveryCompleteEventsHandler.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.TestPlatform.VsTestConsole.TranslationLayer
+{
+    using System;
+    using System.Collections.Generic;
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client.Interfaces;
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel.Logging;
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public class DiscoveryCompleteEventsHandler : ITestDiscoveryEventsHandler2
```

Comment:
Suggestion : We have implemented a new interface `ITestDiscoveryEventsHandler2`, should the clients just move to new implementation, when ever they upgrade to latest version of TestPlatform?

Reply:
Discussed offline. This is resolved.